src/index.js

```diff
@@ -1,2 +1,29 @@
-console.log("I am here");
-console.log("I am here Ramin");
\ No newline at end of file
+import renderHome from './pages/home';
+import renderContact from './pages/contact';
+import renderMenu from './pages/menu';
+import renderNav from './components/shared/header';
+import events from './functions/events';
+import './style/style.css';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function renderPage(currentPage) {
```

Comment:
- [ ] It uses ES6 syntax

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png)

Let's convert this to an arrow function.

Reply:
Done.